### PR TITLE
Adds Hemoglobin Iced Tea, a drink for hemophages

### PR DESCRIPTION
**Pull request by CydiaButt13**
## About The Pull Request

I play hemophages sometimes and we're constantly low on blood unless we're bugging genetics or medical or the bar. So just how synths have a welding fuel drink, I'm making hemos a blood drink. Sprite by me.

## Why It's Good For The Game

This is great for hemos, bloodsuckers, and in a pinch when someone is low on blood, I suppose.

## Proof Of Testing

Took a dumb little video:

https://github.com/user-attachments/assets/13bedf27-a630-4467-b85d-1c7d6cc5d05e

summary>Screenshots/Videos</summary>

## Changelog

:cl:
add: Added sprite for blood_tea
add: Added vending machine option for blood_tea
/:cl:

### CODE REVIEW

**Comment by Genevesh** [comment]
They're using light mode. How do I downvote someone's PR?

**Review by Majkl-J — CHANGES_REQUESTED** [review]
Needs to be modularized, and the icon moved

**Comment by CydiaButt13** [comment]
> They're using light mode. How do I downvote someone's PR?

you cant stop me

**Comment by CydiaButt13** [comment]
okay we're modularized

**Review by Majkl-J — CHANGES_REQUESTED** [review]

File: tgstation.dme
[the remark this answers is not in the document]
Comment (CydiaButt13) [reply tgstation.dme:1]: Ah, it appears some of my changes just reverted there yea, my bad. Hold on

File: modular_zubbers/code/modules/reagents/reagent_containers/cups/soda.dm [hunk 1]
```diff
@@ -0,0 +1,8 @@
+//Soda Modular File for Bubber
+////Add your custom sodas here, use this first one as a reference if you want
+
+/obj/item/reagent_containers/cup/soda_cans/blood_tea
+	name = "Hemoglobin Iced Tea"
+	desc = "Kinda like that rich golfer, but it's actually blood!"
+	icon_state = "modular_zubbers/icons/obj/drinks/soda.dmi"
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/reagents/reagent_containers/cups/soda.dm:7]:
```suggestion
	icon = "modular_zubbers/icons/obj/drinks/soda.dmi"
	icon_state = "blood_tea"
```

**Review by Majkl-J — CHANGES_REQUESTED** [review]

**Review by Majkl-J — APPROVED** [review]

**Review by Majkl-J — APPROVED** [review]

**Pull request merged**